core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -111,17 +115,29 @@ public MetricsSystem(String instance, MetricsConfig metricsConfig) {
     mMetricsConfig = metricsConfig;
   }
 
-  /***
-   * Gets the {@link ServletContextHandler} of the metrics servlet.
+  /**
+   * @return the MetricsServlet sink
+   */
+  public Sink getMetricsServlet() {
```

Comment:
It feels like MetricsServlet shouldn't be understood by things in `common` since it is a concept in `servers` now. Is the motivation to name it this to avoid confusion when doing something like `(MetricsServlet) mc.getSink()`?